tests/rest_api/conftest.py

```diff
@@ -154,6 +175,22 @@ def check(user_id, jid):
             is_task_staff(user_id, jobs[jid]['task_id'])
     return check
 
+@pytest.fixture(scope='module')
+def is_issue_staff(issues, jobs, assignee_id):
+    @ownership
+    def check(user_id, issue_id):
+        return user_id == issues[issue_id]['owner']['id'] or \
+            user_id == assignee_id(issues[issue_id]) or \
+            user_id == assignee_id(jobs[issues[issue_id]['job']])
+    return check
+
+@pytest.fixture(scope='module')
+def is_issue_admin(issues, jobs, is_task_staff):
+    @ownership
+    def check(user_id, issue_id):
+        return is_task_staff(user_id, jobs[issues[issue_id]['job']]['task_id'])
+    return check
+
```

Comment:
Why you decided to separate `issue_admin` and `issue_staff` in such way? Accorging [rego rules](https://github.com/openvinotoolkit/cvat/blob/93ccf2177f560d037fdfad732b98292abfec5944/cvat/apps/iam/rules/issues.rego#L97-L115): if some user is a `issue_admin` then this user automaticly is a `issue_staff`